Form-Controls/index.html

```diff
@@ -1,28 +1,152 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <img src="./img/logo.png" class="logo" alt="brand logo" />
+      <h1>Product Pick</h1>
+    </header>
+    <main>
+      <form>
+        <!-- write your html here-->
+        <!--  1. What is the customer's name? I must collect this data, and validate it. But what is a valid name? I must decide something.
+              2. What is the customer's email? I must make sure the email is valid. Email addresses have a consistent pattern.
+              3. What colour should this t-shirt be? I must give 3 options. How will I make sure they don't pick other colours?
+              4. What size does the customer want? I must give the following 6 options: XS, S, M, L, XL, XXL
+              5. When do they want the t-shirt to be delivered? I must collect a date and make sure that date is in the next four weeks. How will I do this? How will I make sure there are no mistakes about the date?-->
+        <div class="formArea">
+          <div class="groupInptC">
+            <label for="nameInpt">Full Name :</label>
+            <input type="text" name="name" id="nameInpt" required />
+          </div>
+          <div class="groupInptC">
+            <label for="emailInpt">E-mail :</label>
+            <input type="email" name="email" id="emailInpt" required />
```

Comment:
This input box requires an @ which is good, but it doesnt require a full stop and .com after. How can you fix this? Use the example I gave above to fix this!